Gemfile

```diff
@@ -0,0 +1,28 @@
+source 'https://rubygems.org'
+
+ruby '3.3.1'
+
+gem 'bootsnap', require: false
+gem 'dotenv-rails', groups: %i[development test]
+gem 'importmap-rails'
+gem 'jbuilder'
+gem 'pg', '~> 1.1'
+gem 'puma', '>= 5.0'
+gem 'rails', '~> 7.1.3', '>= 7.1.3.4'
+gem 'sprockets-rails'
+gem 'stimulus-rails'
+gem 'turbo-rails'
+gem 'tzinfo-data', platforms: %i[windows jruby]
+
+group :development, :test do
+  gem 'rspec-rails'
```

Comment:
Do we need rspec in development?

Reply:
@SKH-WTAG Though we don't need rspec in development, but if we do use in development it slightly increase load time in development environment, And without using rspec in the development rails generator won't automatically create corresponding rspec test files for our models, controllers etc. We have to manually create test files.